web/src/components/filters-toolbar.tsx

```diff
@@ -334,11 +330,17 @@ export const FiltersToolbar: React.FC<{
             onChange={setFilterValue}
             onKeyPress={e => e.key === 'Enter' && manageFilters()}
             value={selectedFilterValue}
+            ref={searchInputRef}
+            id="search"
           />
         );
     }
   };
 
+  const hasFilterValue = React.useCallback(() => {
+    return filters?.find(f => f?.values?.length) ? true : false;
```

Comment:
This should be equivalent:

```suggestion
    return filters?.some(f => f?.values?.length);
```